app/src/main/res/layout-port/schedule.xml

```diff
@@ -13,12 +13,10 @@
             android:background="@color/schedule_room_name_header_background"
             android:layout_height="wrap_content">
 
-        <TextView
-                android:layout_height="wrap_content"
-                android:layout_width="@dimen/time_width"
-                android:text=""
-                android:textColor="@color/schedule_room_name_header_text"
-                tools:text="Ada"/>
+        <Space
+            android:layout_width="@dimen/time_width"
+            android:layout_height="wrap_content"
+            tools:layout_height="22dp" />
```

Comment:
why no more `tools:text` ?

Reply:
There is no text shown in the view. It is just the black spacer in the top right corner: top of the time column, left of the room names. Setting the tools:text before was wrong.

Reply:
![image](https://user-images.githubusercontent.com/144518/80390222-6f394c80-88ac-11ea-80c5-9fdf5b6694d5.png)